pom.xml

```diff
@@ -95,7 +95,7 @@
         <module>example-library</module>
         <module>example-runner</module>
         <module>performance</module>
-        <module>deploy-example-jars</module>
+        <module>deployment</module>
     </modules>
```

Comment:
Does this change mean we can remove the repository for deploy-example-jars?

Reply:
Once merged, yes